_test/CMakeLists.txt

```diff
@@ -45,16 +45,17 @@ set(ENV_VARIABLES
 )
 
 foreach(_test_dir ${TEST_DIRECTORIES})
-    set(TEST_NAME "Matlab.${_test_dir}")
     if("${_test_dir}" STREQUAL "test_sqw")
         set(TEST_TIMEOUT_LENGTH "1500")  # 25 minutes
-    elseif("${_test_dir}" STREQUAL "test_gen_sqw_workflow")
+    elseif("${_test_dir}" STREQUAL "test_gen_sqw_workflow/test_gen_sqw_accumulate_sqw_parpool")
```

Comment:
this change reverts all the other tests in `test_gen_sqw_workflow` to the default 960sec/16min. Is that intentional?

Reply:
This block wasn't actually doing anything before since it was testing for an exact string match with `test_gen_sqw_workflow`, which none of our tests satisfy. 

It must have been leftover from before we split the `gen_sqw_workflow` tests up.